### ISSUE TITLE
How do I add this function to the VGG16(D,D) bilinear model?

### ISSUE THREAD

**Issue by JUSTDODoDo** [ronghanghu/tensorflow_compact_bilinear_pooling#6]
Dear ronghanghu：
        Thank you for your work，I tried to call this function to reproduce the paper, but the loss (cost function) has been very large during the training, and there is no tendency to decrease. It may be due to divergence. Can you help me see what is wrong?
       self.cbp = compact_bilinear_pooling_layer(self.conv5_3, self.conv5_2, 16000, sum_pool=True)
      In the implementation process, I use Vgg 16 conv5_2, conv5_3 as the input of bottom1 and bottom2, and then pass the obtained self.cbp directly to the full-connect layer softmax classifier.
      But the loss of the training set and the validation set has been very large and can't converge. Can you tell me if there are some missing steps in the function process? I use random gradient descent to optimize the final prediction value and the cross entropy of the label. The batchsize is 32.

**Comment by JUSTDODoDo** [comment]
This work has been bothering me for many days and I look forward to your reply, thank you!

**Comment by zoupei123456** [comment]
Hello, when I run compact_bilinear_pooling_test, the following error occurs. Can you tell me what is going on?

File "D:/jupyter/tensorflow_compact_bilinear_pooling-master/compact_bilinear_pooling_test.py", line 5, in <module>
    from compact_bilinear_pooling import compact_bilinear_pooling_layer
  File "D:\jupyter\tensorflow_compact_bilinear_pooling-master\compact_bilinear_pooling.py", line 6, in <module>
    from sequential_fft import sequential_batch_fft, sequential_batch_ifft
  File "D:\jupyter\tensorflow_compact_bilinear_pooling-master\sequential_fft\__init__.py", line 1, in <module>
    from .sequential_batch_fft_ops import sequential_batch_fft, sequential_batch_ifft
  File "D:\jupyter\tensorflow_compact_bilinear_pooling-master\sequential_fft\sequential_batch_fft_ops.py", line 10, in <module>
    'build/sequential_batch_fft.so'))
  File "C:\Users\window10\AppData\Local\Continuum\Anaconda3\lib\site-packages\tensorflow\python\framework\load_library.py", line 56, in load_op_library
    lib_handle = py_tf.TF_LoadLibrary(library_filename)
tensorflow.python.framework.errors_impl.NotFoundError:libtensorflow_framework.os:cannot open shared object file:No such file or directory
